tests/test_infer/test_inference_engine.py

```diff
@@ -44,7 +40,7 @@ def check_inference_engine(use_engine=False, prompt_template=None):
     top_k = 50
 
     if use_engine:
-        inference_config = InferenceConfig(max_output_len=output_len, prompt_template=prompt_template)
+        inference_config = InferenceConfig(max_output_len=output_len, prompt_template=prompt_template, dtype="fp32")
```

Comment:
Why update this to "fp32"?

Reply:
after switching to cuda kernel implementation, now we can not pass the precision test for whole model inference under fp16, thus we need switch to fp32, here dtype will determine the exact data type of model parameters and need to be set to fp32

Reply:
after switching to cuda kernel implementation, now we can not pass the precision test for whole model inference under fp16, thus we need switch to fp32, here dtype will determine the exact data type of model parameters and need to be set to fp32